intdataplane/int_dataplane.go

```diff
@@ -463,16 +468,24 @@ func (d *InternalDataplane) loopUpdatingDataplane() {
 		}
 
 		if datastoreInSync && d.dataplaneNeedsSync {
+			log.Info("Applying updates to dataplane.")
 			applyStart := time.Now()
 			d.apply()
-			applyEnd := time.Now()
-			if applyEnd.After(applyStart) {
+			applyTime := time.Since(applyStart)
+			if applyTime > 0 {
 				// Avoid a negative interval in case the clock jumps.
-				histApplyTime.Observe(applyEnd.Sub(applyStart).Seconds())
+				histApplyTime.Observe(applyTime.Seconds())
 			}
 			if d.dataplaneNeedsSync {
 				countDataplaneSyncErrors.Inc()
 			}
+			log.WithField("msecToApply", applyTime.Seconds()/1000.0).Info(
```

Comment:
Note: I'm forcing these to msec/seconds to make Casey's parsing easier.  Without that, they'll pretty-print in an awkward-to-parse way.